build.sh

```diff
@@ -169,6 +215,33 @@ done
 custom_depenguin_installdir="customfiles/root"
 mkdir -p "$custom_depenguin_installdir"
 
+# setup correct files to copy in based on version
+if [ "$MYVERSION" = "13.2" ]; then
+	if [ -f "$MYCUSTOMDIR/13_2_depenguin_bsdinstall.sh" ]; then
+		cp -f "$MYCUSTOMDIR/13_2_depenguin_bsdinstall.sh" "$MYCUSTOMDIR/depenguin_bsdinstall.sh"
+		chmod +x "$MYCUSTOMDIR/depenguin_bsdinstall.sh"
+	else
+		echo "Missing $MYCUSTOMDIR/13_2_depenguin_bsdinstall.sh"
+	fi
+	if [ -f "$MYCUSTOMDIR/13_2_INSTALLERCONFIG.sample" ]; then
+		cp -f "$MYCUSTOMDIR/13_2_INSTALLERCONFIG.sample" "$MYCUSTOMDIR/INSTALLERCONFIG.sample"
+	else
+		echo "Missing $MYCUSTOMDIR/13_2_INSTALLERCONFIG.sample"
+	fi
+elif [ "$MYVERSION" = "14.0" ]; then
+	if [ -f "$MYCUSTOMDIR/14_0_depenguin_bsdinstall.sh" ]; then
+		cp -f "$MYCUSTOMDIR/14_0_depenguin_bsdinstall.sh" "$MYCUSTOMDIR/depenguin_bsdinstall.sh"
+		chmod +x "$MYCUSTOMDIR/depenguin_bsdinstall.sh"
+	else
+		echo "Missing $MYCUSTOMDIR/14_0_depenguin_bsdinstall.sh"
+	fi
+	if [ -f "$MYCUSTOMDIR/14_0_INSTALLERCONFIG.sample" ]; then
+		cp -f "$MYCUSTOMDIR/14_0_INSTALLERCONFIG.sample" "$MYCUSTOMDIR/INSTALLERCONFIG.sample"
+	else
+		echo "Missing $MYCUSTOMDIR/14_0_INSTALLERCONFIG.sample"
+	fi
+fi
+
```

Comment:
Very copy-and-pasty, maybe it could be something like (untested)

```
VER_PREFIX=$(echo "$MYVERSION" | sed "s/\./_/")
```

and then make use of VER_PREFIX in all the copy commands and error messages.